graphql/resolvers.go

```diff
@@ -3019,6 +3019,9 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 	if flags.PluginAdminPageDisabled {
 		return model.IsPerfEnabledForProject(*obj.ProjectId), nil
 	} else {
+		if !evergreen.IsFinishedTaskStatus(utility.FromStringPtr(obj.Status)) {
+			return false, nil
+		}
```

Comment:
This can be moved up to before line 3015 so we can avoid doing any work for this resolver if we know there won't be any perf results.